internal/cli/run.go

```diff
@@ -106,3 +113,27 @@ func getChangesReportURL(out string) string {
 
 	return ""
 }
+
+func processRegistryTags() []string {
```

Comment:
Are we supposed to be handling the source prefixes here?

Reply:
Nope. That has to happen where the tags are applied.

Reply:
https://github.com/speakeasy-api/speakeasy/pull/660